src/tests/signup.test.js

```diff
@@ -0,0 +1,41 @@
+import Chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+Chai.use(chaiHttp);
+Chai.should();
+
+const user = {
```

Comment:
@AbdoulNuru I think you could put this mock data in the mock folder
Put this 
```suggestion
const user = {
  firstName: 'Nuru',
  lastName: 'Abd',
  email: 'abdoulniyigena@gmail.com',
  password: 'nuru1234'
};
```
in file
   ``
tests/mockData/user.mock.js `` like that.

Reply:
@hakoemmy Thank you, it is done